### PR TITLE
[DOCS-6593] - Alfresco Digital Workspace 2.7

### PR DESCRIPTION
**Pull request by Amy-Hyland**

### CODE REVIEW

**Review by lilustig — APPROVED** [review]
LGTM

File: digital-workspace/latest/support/index.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Supported Platforms
```
Comment (anxumalo) [inline digital-workspace/latest/support/index.md:2]: I think this file is missing the ACS 7.2 addition - unless it's in another branch?

Comment (Amy-Hyland) [reply digital-workspace/latest/support/index.md:2]: I think I've updated it appropriately now for as 7.2.

**Review by anxumalo — APPROVED** [review]
LGTM

**Review by lilustig — APPROVED** [review]
LGTM

**Pull request merged**